css/styles.css

```
@@ -4,27 +4,85 @@
   margin: 0;
 }
 
+.header {
+  background-color: none;
+  padding: 0;
+  text-align: center;
+}
+
+.header img {
+  max-width: 100%;
+  height: auto;
+}
+
 body {
-  font-family: Arial, Helvetica, sans-serif;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
   font-size: 20px;
+  background-color: #f1fffd;
 }
 
 .question-card {
   display: flex;
   flex-direction: column;
-  border: 1px solid red;
+  border: groove #e95151;
+  border-radius: 10px;
   padding: 20px;
-  margin: 10px;
-  background-color: skyblue;
+  margin: 20px;
+  margin-bottom: 20px;
+  background-color: #ffdadd;
+  position: relative;
 }
 
-.question-number {
-  color: red;
-  text-align: center;
+.bookmark-corner {
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  right: -20px;
+  top: -20px;
+  background-color: none;
+}
+
+.tags {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  padding: 20px;
+  justify-content: center;
 }
 
 .question-text {
-  color: green;
+  color: #40a594;
   text-align: center;
   margin-block: 20px;
 }
+
+button {
+  background-color: white;
+  color: black;
+  padding: 5px 5px;
+  font-size: 16px;
+  border: 1px solid;
+  border-radius: 10px;
+  cursor: pointer;
```

i don't know if you wanted to change the width of the button, but it should work fine with

```suggestion
  width: ...px ;
```